spec/models/concerns/with_qualifications_spec.rb

```diff
@@ -22,6 +22,14 @@
     end
   end
 
+  describe '#qualifications_description' do
+    context 'no qualification present' do
+      subject { build(:course, qualification: nil) }
+
+      its(:qualifications_description) { should eq '' }
```

Comment:
just wondering is should there be a default qualifications?

Reply:
@fofr

Reply:
There's no default. A course must always have a qualification.

Reply:
should we enforce that @misaka ?

Reply:
Similar to `SubjectMapperService.get_subject_list`, the change to `WithQualifications`'s `qualifications_description` was done just to facilitate returning a new course for the "create course" wizard. It's not valid to have a course have no qualifications, but that should be caught by a validation since until we persist to the db, we want to allow invalid course objects to be instantiated so that we can help out the wizard.